History.md

```diff
@@ -1,9 +1,11 @@
 # Liquid Change Log
 
-## 5.3.0 (unreleased)
+## 5.3.0 2022-03-22
 
 ### Fixes
 * StandardFilter: Fix missing @context on iterations (#1525) [Thierry Joyal]
+* Test under Ruby 3.1 (#1533) [petergoldstein]
```

Comment:
This isn't really a change from the user's perspective.  The code already worked in ruby 3.1, so this isn't really a fix.  Users of the library generally don't care about tests being added.